Toolchain/CMake/LLVMRuntimesConfig.cmake

```diff
@@ -11,6 +11,10 @@ set(LLVM_ENABLE_RUNTIMES "libcxx;libcxxabi;libunwind" CACHE STRING "")
 set(CMAKE_SYSTEM_NAME SerenityOS CACHE STRING "")
 set(target_triple ${SERENITY_TOOLCHAIN_ARCH}-pc-serenity)
 
+IF(${CMAKE_HOST_SYSTEM_NAME} MATCHES "Darwin")
+    set(CMAKE_BUILD_WITH_INSTALL_RPATH ON CACHE BOOL "")
+ENDIF()
```

Comment:
A bit of a nitpick, please use lowercase `if` and `endif`, and put a space after `if`:
```cmake
if (${CMAKE_HOST_SYSTEM_NAME} MATCHES "Darwin")
    set(CMAKE_BUILD_WITH_INSTALL_RPATH ON CACHE BOOL "")
endif()
```

Reply:
Oops, commented too late

Reply:
oops, didn't notice that ^^'